recipes/userver/all/conanfile.py

```diff
@@ -0,0 +1,685 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, export_conandata_patches, apply_conandata_patches
+from conan.tools.build import cross_building
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
+
+class UserverConan(ConanFile):
+    name = 'userver'
+    description = 'The C++ Asynchronous Framework'
+    topics = ('framework', 'coroutines', 'asynchronous')
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://userver.tech/'
+    license = 'Apache-2.0'
+    package_type = "static-library"
+
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    options = {
+        'fPIC': [True, False],
+        'lto': [True, False],
+        'with_jemalloc': [True, False],
+        'with_mongodb': [True, False],
+        'with_postgresql': [True, False],
+        'with_postgresql_extra': [True, False],
+        'with_redis': [True, False],
+        'with_grpc': [True, False],
+        'with_clickhouse': [True, False],
+        'with_rabbitmq': [True, False],
+        'with_utest': [True, False],
+        'with_testsuite' : [True, False],
+        'namespace': ['ANY'],
+        'namespace_begin': ['ANY'],
+        'namespace_end': ['ANY'],
+    }
+
+    default_options = {
+        'fPIC': True,
+        'lto': True,
+        'with_jemalloc': False, # Disabled dy default due to jemalloc recipe does not support Conan v2
```

Comment:
Jemalloc is now available in Conan 2, feel free to make this True by default if upstream following https://github.com/userver-framework/userver/blob/13ca039791f9dbaa9da36e3e55e86451516dfa7f/CMakeLists.txt#L96 :)

Reply:
I know but I fixed this recipe so many tries that now I have no desire to do it again